app/org/sagebionetworks/bridge/services/ExportService.java

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.bridge.services;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
+
+/** Service that interfaces Bridge Server with Bridge Exporter. */
+public interface ExportService {
+    /** Kicks off an on-demand export for the given study. */
+    void startOnDemandExport(StudyIdentifier studyId) throws JsonProcessingException;
+}
```

Comment:
I only mention it because we removed interfaces for all the other services on the premise that until we have more than one implementation, we don't need the extra complexity.